client/web/compose/src/components/ModuleFields/Configurator/File.vue

```diff
@@ -24,6 +24,19 @@
       />
     </b-form-group>
 
+    <b-form-group
+      :label="$t('kind.file.view.enableWebcamLabel')"
```

Comment:
You can remove the label itself and the label class.
Just the checkbox with its label is enough.